### ISSUE TITLE
Cultivation weired behaviour

### ISSUE THREAD

**Issue by Alfred19** [WarEmu/WarBugs#12941]
Hi,
i rolled a new toon, specced Cultivation and reached 200, grew a few stacks of lv 200 mats.
Could not harvest all of a sudden.
Disabled addons, restarted game - no difference.
Checked other toons with the skill - no problems visible.
Logged to new toon, .respec gathering.
Starting from 0 again, gone leveling run MGR meanwhile.
Screenmessage: Your Culti... 49, harvest... ? harvest harvest harvest harvest harvest... here we go again.

Contact me pls if u need additional info.
cheers

**Comment by sioding** [comment]
I´ve read this about 4 times and do not get the problem. The problem is you do not reach the skillevel 50 ?

**Comment by Alfred19** [comment]
Thx for the reply.
As i said I WAS already on 200 before, problem is it bugs out randomly for no apparent reason.

Actually for some reason it started working again in the meantime,  was able to reach 67 now - but again... i can harvest anymore. Making progress impossible

**Comment by Alfred19** [comment]
Just now i could harvest and regrow 1 out of  2... 2nd plot is unharvestbable.

**Comment by Alfred19** [comment]
both plots locked down again

**Comment by sioding** [comment]
ok got it - tried to reproduce it on my own - no "luck" until now.

**Comment by Alfred19** [comment]
Update it works again, no restar or relog nor lvl up of some sort r88 now.
its just weired its the only toon i exp this so far

**Comment by Alfred19** [comment]
between 200 and .respec one night has passed actually

**Comment by sioding** [comment]
ok - thanks for the informations. I will talk to a dev about this but its kinda hard cause its not reproducable in a serious way. Anyway - have a nice christmas.

**Comment by Alfred19** [comment]
Yep that is what bothers me a bit in this case. Thanks, you too.
